udf/bits28_retention.sql

```diff
@@ -0,0 +1,58 @@
+/*
+
+
+Return a nested struct providing numerator and denominator fields for
```

Comment:
Is this providing a numerator and denominator? It may be better to specify this as e.g. `Return a nested struct providing activity information as a boolean for the standard ...`.